language/en/interface.json

```diff
@@ -644,6 +644,7 @@
 			"waitingFor": "Waiting for %{name}",
 			"turn": "turn",
 			"players": "players",
+			"teamToLoad": "team to load",
```

Comment:
Translation entries should be entire phrases, as some languages use different subject-object or adjective-noun-verb orders.

Reply:
Sure thing, I'll rename them.

Reply:
Expanded translations.
```json
"draftOrderMod": {
	"teamPlacement": "Team Placement",
	"placeYourCom": "Place your commmander",
	"waitingFor": "Waiting for %{name}",
	"waitingForTurn": "Waiting for turn",
	"waitingForPlayers": "Waiting for players",
	"waitingForTeamToLoad": "Waiting for team to load",
	"nextIsPlayer": "Next is %{name}",
	"modeRandom": "Random draft order start position mode activated. Players await their turn to place",
	"modeSkill": "Skill-based draft order start position mode activated. Players await their turn to place",
	"modeFair": "Fair start position mode activated. Please wait until players joins the game before placing"
},
```